cmd/kubeadm/app/phases/certs/certs.go

```diff
@@ -276,6 +295,30 @@ func writeKeyFilesIfNotExist(pkiDir string, baseName string, key *rsa.PrivateKey
 	return nil
 }
 
+func writeCSRFilesIfNotExist(csrDir string, baseName string, csr *x509.CertificateRequest, key *rsa.PrivateKey) error {
```

Comment:
I'd rather put a big comment on top of this one, to prevent people from accidentally calling it.